controllers/admin/AdminPdfController.php

```diff
@@ -72,7 +72,6 @@ public function processGenerateOrderSlipPDF()
             die($this->trans('The order cannot be found within your database.', [], 'Admin.Orderscustomers.Notification'));
         }
 
-        $order->products = OrderSlip::getOrdersSlipProducts($order_slip->id, $order);
```

Comment:
Hi @PululuK why do you say this code is unusable?

Do you mean that the following code is useless as `$order->products` is not used after line 75?

Reply:
@matks exact !